DigitalLearningSolutions.Data/Models/ProgressCompletionData.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace DigitalLearningSolutions.Data.Models
+{
+    using System.Text.RegularExpressions;
+
+    public class ProgressCompletionData
+    {
+        public int CentreId { get; set; }
+        public string CourseName { get; set; }
+        public string? AdminEmail { get; set; }
+        public int SessionId { get; set; }
+
+        public string GetCourseNameWithoutAverageTimeInfo()

```

Comment:
Does this actually do anything? The SQL query that gets these doesn't seem to have a time part on it?

Reply:
The way I understood it from the ticket was that sometimes admins will put the average time info at the end of the CustomisationName, but I could be mistaken. Maybe @SteveJacksonSoft can clarify?

Reply:
Ah OK, in that case it will probably, but not guaranteed pick it up. At worst this just doesn't do anything though.

Reply:
Did this get resolved?

Reply:
Not yet, see also https://github.com/TechnologyEnhancedLearning/DLSV2/pull/1084#discussion_r858523605

Reply:
Answered there.